config/samples/python_asyncio_example_loadtest.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: e2etest.grpc.io/v1
```

Comment:
I think it is okay this time, you might want to avoid mixing things up in a single PR in the future. For example, this PR is dedicated for adding prebuilt images for python, it made more or less scene to also include prebuilt images for python_asyncio since they are both included by word python. But for adding python_asyncio_example_loadtest.yaml it makes more scene to have its own PR. This way, when we select PRs to do a release,  by reading the name of PR we have a basic understanding of what it is, also it gives us options to include the change this time or next. :)

Reply:
Thanks, this definitely make sense. Done.